data/brands/amenity/cafe.json

```diff
@@ -3534,6 +3535,27 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "太平洋咖啡 Pacific Coffee",
+      "id": "pacificcoffee-2e3d52",
+      "locationSet": {
+        "include": ["cn", "my", "sg"],
```

Comment:
This bilingual entry should include "hk" and "mo"

Reply:
Updated.